carina-webdriver/src/main/java/com/qaprosoft/carina/core/foundation/utils/android/IAndroidUtils.java

```diff
@@ -581,5 +583,33 @@ default public void checkTimeout(long startTime){
             throw new NoSuchElementException("Scroll timeout has been reached..");
         }
     }
+    
+    /**
+     * Check running apk by appPackage or appActivity from Capabilities
+     *
+     * @return boolean
+     */
+    default boolean isAppRunning() {
+        Capabilities capabilities = ((RemoteWebDriver) castDriver()).getCapabilities();
+        return isAppRunning(String.valueOf(capabilities.getCapability("appPackage"))) || isAppRunning(String.valueOf(capabilities.getCapability("appActivity")));
+    }
+    
+    /**
+     * Check running apk 
+     *
+     * @param apk String
+     * @return boolean
+     */
+    default boolean isAppRunning(String apk) {
+        AndroidService androidService = AndroidService.getInstance();
```

Comment:
remove reference to AbdroidService please. All functionality step-by-step should be moved to interface

Reply:
done

Reply:
done